db/models/users.model.js

```diff
@@ -0,0 +1,50 @@
+const { DataTypes } = require("sequelize");
+module.exports = (sequelize) => {
+  const User = sequelize.define("users", {
+    id: {
+      type: DataTypes.UUID,
+      unique: true,
+      defaultValue: DataTypes.UUIDV4,
+    },
+    email: {
+      type: DataTypes.STRING,
+      unique: true,
+      allowNull: false,
+      primaryKey: true,
+    },
+    password: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    role: {
+      type: DataTypes.STRING(10),
+      allowNull: false,
+    },
+    firstname: {
+      type: DataTypes.STRING(50),
+      allowNull: false,
+    },
+    lastname: {
+      type: DataTypes.STRING(50),
+      allowNull: false,
+    },
+    gender: {
+      type: DataTypes.STRING(10),
+      allowNull: false,
+    },
+    dob: {
+      type: DataTypes.DATEONLY,
+      allowNull: false,
+    },
+    isActive: {
+      type: DataTypes.BOOLEAN,
+      defaultValue: false,
+    },
+    isVerified: {
+      type: DataTypes.BOOLEAN,
+      defaultValue: false,
+    },
+  });
```

Comment:
Hope you have confirmed the created date and last updated date are auto generated?